prow/cmd/deck/main.go

```diff
@@ -1495,6 +1498,52 @@ func HandleGitHubLink(githubHost string, secure bool) http.HandlerFunc {
 	}
 }
 
+// HandleGenericProviderLink returns link based on different providers.
+func HandleGenericProviderLink(githubHost string, secure bool) http.HandlerFunc {
```

Comment:
Please add `TestHandleGenericProviderLink` to main_test.go

Reply:
done